trino/base/hive-metastore-dc.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hive-metastore
+  labels:
+    instance: dev-presto
+    role: hive-metastore
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      instance: dev-presto
+      role: hive-metastore
+  template:
+    metadata:
+      labels:
+        instance: dev-presto
+        role: hive-metastore
+    spec:
+      containers:
+        - resources:
+            requests:
+              cpu: 1
+              memory: 4Gi
+            limits:
+              cpu: 1
+              memory: 4Gi
+          terminationMessagePath: /dev/termination-log
+          name: hive-metastore
+          command:
+            - /opt/bin/start-hive-metastore.sh
+          livenessProbe:
+            exec:
+              command:
+                - bash
+                - '-c'
+                - netstat -lpn | grep 9083
+            initialDelaySeconds: 30
+            timeoutSeconds: 30
+            periodSeconds: 30
+            successThreshold: 1
+            failureThreshold: 5
+          env:
+            - name: HIVE_METASTORE_JDBC_URL
+              value: >-
+                jdbc:postgresql://trino-db.$(namespace).svc.cluster.local:5432/trino
```

Comment:
Have you tested to see if this works it you just do `jdbc:postgresql://trino-db:5432/trino`? I think it may work without the fully qualified service hostname, and that would simplify this substantially.